src/tests/suite/SConscript

```diff
@@ -97,7 +29,7 @@ def scons():
                                c_files + daos_test_tgt,
                                LIBS=['daos_common'] + libraries)
 
-    newenv = configure_cmocka(denv.Clone())
```

Comment:
Why isn't this needed anymore?

Reply:
I tested - none of the distros we have in CI use a version of cmocka with the bug we're testing for.